packages/bridging/src/providers/near-intents/NearIntentsBridgeProvider.ts

```diff
@@ -0,0 +1,313 @@
+import { getGlobalAdapter, setGlobalAdapter } from '@cowprotocol/sdk-common'
+import { ETH_ADDRESS } from '@cowprotocol/sdk-config'
+import { CowShedSdk } from '@cowprotocol/sdk-cow-shed'
+import { OrderKind } from '@cowprotocol/sdk-order-book'
+import { QuoteRequest } from '@defuse-protocol/one-click-sdk-typescript'
+import { BridgeStatus } from '../../types'
+import { concat, getAddress, hexToBytes, isHex, keccak256, recoverAddress } from 'viem'
```

Comment:
Unfortunatelly, we cannot import directly from viem, we have to use getGlobalAdapter().utils instead

Reply:
https://github.com/cowprotocol/cow-sdk/pull/478#discussion_r2510140007